features/base/test/test_umask.py

```diff
@@ -16,5 +16,15 @@
 
 # Run the test unit to perform the
 # final tests by the given artifact.
-def test_umask(client, file, dict):
```

Comment:
Very nice pr description, but for visibility reasons I'd suggest to add a comment here to explain why it's disabled on gcp (for now). Will make it easier to come back to it later when we know more.

Reply:
addressed this in latest commit